workflows/tf-object-detection-training/main.py

```diff
@@ -43,7 +43,7 @@ def main(params):
         '/mnt/src/tf/research/object_detection/legacy/train.py',
         '--train_dir=/mnt/output/',
         '--pipeline_config_path=/mnt/output/pipeline.config',
-        '--num_clones={}'.format(params['num-clones'])
+        '--num_clones={}'.format(params['num_clones'])
```

Comment:
@javoweb we'd need to also support the old keys (with the `-`) for the existing params to retain backwards compatibility.

Reply:
Yes, I'm doing it on a new function  `legacy_params_compatibility` I'm parsing old keys to the newer ones. But the rest of the code use the new keys with the `_` only.

Reply:
Ah, missed that. Looks good!